serve.py

```diff
@@ -67,30 +67,45 @@
 CORS(app)
 
 
+def make_header(url):
+    import time
+    gmtnow = time.strftime("%a, %d %b %Y %I:%M:%S %p %Z", time.gmtime())
+    header = {"location": url,
+              "content-type": "text/plain;charset=UTF-8",
+              "content-length": 0,
+              "date": gmtnow,
+              "alt-svc": "clear"}
+    return header
+
+
 @app.listener("before_server_start")
 async def init(app, loop):
     global sem
     sem = asyncio.Semaphore(100)
 
 
-@app.route("/")
+@app.route("/", methods=['GET'])
 async def main(request):
+    if request.method == "HEAD":
+        pass
     return response.redirect("https://gui.dandiarchive.org/")
 
 
-@app.route("/about")
+@app.route("/about", methods=['GET'])
 async def about(request):
+    if request.method == "HEAD":
```

Comment:
I am not familiar with this framework, wondering - if methods only GET, why to check here for head?

Reply:
see the decorator above.

Reply:
yes, and it only lists GET in the `methods`, so I assumed that it would simply not get into `about` for HEAD method, so saw no point in this `==` here.

Reply:
i think github is not showing the diff properly. take a look at the file itself. those two functions have `GET` and `HEAD`

Reply:
Checked, diff is ok. Anyways - it is just a pass statement, so what is the point? ;-)

Reply:
sorry that was a mistake will fix in master. i had put a bunch of those in, but then removed